src/components/proposalActions.jsx

```diff
@@ -39,7 +39,7 @@ import { earlyExecuteMinionType } from '../utils/minionUtils';
 
 const MotionBox = motion(Box);
 
-const ProposalVote = ({
+const ProposalActions = ({
```

Comment:
the file is called proposalActions, the component was named ProposalVote, I am actually suprissed this loaded in the UI before, as there is a reference to `<ProposalActions .../>` iin the app. Does react only base loading on the file name? 

or maybe this is because its the default export - thats probably it... anyways updating the naming so its more trackable throughout the App.